src/core/net/srp_client.hpp

```diff
@@ -410,6 +410,27 @@ class Client : public InstanceLocator, private NonCopyable
      */
     void SetCallback(Callback aCallback, void *aContext);
 
+    /**
+     * This method gets the TTL used in SRP update requests.
+     *
+     * Note that this is the TTL that would be requested by the SRP client. Server may choose to accept a different TTL.
+     *
+     * @returns The TTL (in seconds).
```

Comment:
Suggest mentioning that value 0 is a special case and causes TTL to follow lease time

Reply:
This will always return the TTL value used in SRP Update requests. If `mTtl` is set to zero, this will return `mLeaseInterval`.